### PR TITLE
feat: expose revideo project as web service

### PR DESCRIPTION
**Pull request by justusmattern27**

### CODE REVIEW

File: packages/serve/package.json [hunk 1]
```diff
@@ -16,12 +16,12 @@
...
   "devDependencies": {
-    "@types/express": "^4.17.21",
     "typescript": "^5.4.3"
   },
   "dependencies": {
     "@revideo/renderer": "^0.2.3",
     "commander": "^12.0.0",
-    "uuid": "^9.0.1"
+    "uuid": "^9.0.1",
+    "@types/express": "^4.17.21"
```
Comment (hkonsti) [inline packages/serve/package.json:25]: types go into devDependencies

File: packages/serve/src/server.ts [hunk 2]
```diff
@@ -0,0 +1,114 @@
+import {renderVideo} from '@revideo/renderer';
+import axios from 'axios';
+import express from 'express';
+import {promises as fs} from 'fs';
+import path from 'path';
+import {v4 as uuidv4} from 'uuid';
+
+const App = express();
```
Comment (hkonsti) [inline packages/serve/src/server.ts:8]: minor nitpick, App and Port should be lowercase imo

File: packages/serve/src/server.ts [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+  setTimeout(
+    async () => {
+      try {
+        await fs.unlink(filePath);
+        console.log(`Successfully deleted file: ${filePath}`);
+      } catch (error) {
+        console.error(`Error deleting file ${filePath}: ${error}`);
+      }
+
+      app._router.stack = app._router.stack.filter((layer: any) => {
```
Comment (hkonsti) [inline packages/serve/src/server.ts:103]: I don't understand what this does and why this is needed

**Review by hkonsti — COMMENTED** [review]
I think it'd be cooler if we called the new package `cli` instead of `serve` because then we can add much more functionality in the future. What do you think?

File: packages/serve/src/server.ts (hunk 3 above)
Comment (justusmattern27) [reply packages/serve/src/server.ts]: fixed this using the approach we discussed

File: packages/serve/package.json (hunk 1 above)
Comment (justusmattern27) [reply packages/serve/package.json]: done

**Comment by justusmattern27** [comment]
I agree with renaming to cli. Just did that

**Review by hkonsti — APPROVED** [review]
looks good 👍

**Pull request merged**